RapidReact/src/main/java/frc/robot/Robot.java

```diff
@@ -217,7 +217,7 @@ public void autonomousInit() {
               .executeDrivePath("Main P2 Ball", 2.0) //Skip terminal, go straight to the second ball
               .executeAction(AutonomousCommand.SubsystemAction.IntakeToggleAction, 2.0) //Turn off the intake after getting the ball
               .executeDrivePath("Main P3") //Drive to the base of the hub
-              .executeAction((d, i, s) -> s.shootTop()) //Shoot
```

Comment:
Don't you need to have it also run the shoot action after spinning up? Also in line 238.

Reply:
@Calculus-Master can you update auto to use this new system?